ansible/roles/host-ocp4-installer/templates/install-config.yaml.j2

```diff
@@ -7,6 +7,9 @@ fips: {{ ocp4_fips_enable | bool | to_json }}
 controlPlane:
   name: master
   hyperthreading: Enabled
+{% if openshift_architecture | default('x86_64') is match('arm64') %}
```

Comment:
Let's put in the role defaults:
```
openshift_architecture: x86_64
```

Then cleanup all the repeated `| default(x86_64` ?

Reply:
Actually changed the variable to ocp4_architecture_cluster. Seems more consistent with variable naming. Will also allow us to add an ocp4_architecture_bastion in the future should we want to support ARM bastions.